circle.yml

```diff
@@ -862,6 +863,31 @@ jobs:
             chmod +x codecov
             ./codecov -c
 
+# ------------------------------------------------------------------------------
+  linux-doxygen:
+    docker:
+      - image: mbgl/linux-gcc-5:54f59e3ac5
+    resource_class: large
+    working_directory: /src
+    environment:
+      LIBSYSCONFCPUS: 2
+      JOBS: 2
+      BUILDTYPE: Debug
+      WITH_EGL: 1
+      WITH_COVERAGE: 1
+    steps:
+      - install-dependencies
+      - run:
+          name: Install doxygen
+          command: apt update && apt install -y doxygen
+      - build-linux
```

Comment:
Do we need to build to get the doxygen coverage?

Reply:
Not really, but the doxygen documentation is being generated by cmake so the build needs to be configured. Doing only that is a bit tricky with the current state of the build system. I bundled the coverage check with the codecov job to save one build.

Reply:
Actually, scratch that. We need doxygen to be installed, so it's probably better to have the commands to install it in a separate job.